ts/Dashboards/Board.ts

```diff
@@ -623,6 +623,39 @@ class Board implements Serializable<Board, Board.JSON> {
         };
     }
 
+    /**
+     * Get the board's options.
```

Comment:
That is a public method. IMO needs better description what it returns, what are the limitations and what you can do with it ;)